bodhi/server/scripts/approve_testing.py

```diff
@@ -60,16 +46,23 @@ def main(argv=sys.argv):
     pushed to stable.
 
     This function is the entry point for the bodhi-approve-testing console script.
-
-    Args:
-        argv (list): A list of command line arguments. Defaults to sys.argv.
     """
-    logging.basicConfig(level=logging.ERROR)
-
-    if len(argv) != 2:
-        usage(argv)
-
-    settings = get_appsettings(argv[1])
+    parser = argparse.ArgumentParser(
+        description='Comment on updates that are eligible to be pushed to stable.'
+    )
+    parser.add_argument('config_uri', help='Path to the configuration file to use')
+    parser.add_argument(
+        '--debug', dest='debug', action='store_true', default=False,
+        help='Increase the verbosity level of the output')
+
+    args = parser.parse_args(argv)
+
+    if args.debug:
+        logging.basicConfig(level=logging.INFO)
```

Comment:
This feels odd, that `--debug` sets the log level to `INFO`. Should this be a `--verbose` flag instead?

Reply:
wfm

Reply:
Heh, if anybody is ever confused about this, I'll send them your way. :stuck_out_tongue:

Reply:
I've adjusted it! :D